extension/js/common/composer/composer-draft.ts

```diff
@@ -0,0 +1,239 @@
+/* © 2016-2018 FlowCrypt Limited. Limitations apply. Contact human@flowcrypt.com */
+
+'use strict';
+
+import { ComposerAppFunctionsInterface } from './interfaces/composer-app-functions.js';
+import { Ui, Env } from '../browser.js';
+import { Xss } from '../platform/xss.js';
+import { Mime } from '../core/mime.js';
+import { Buf } from '../core/buf.js';
+import { Pgp, PgpMsg } from '../core/pgp.js';
+import { Api, AjaxError } from '../api/api.js';
+import { BrowserMsg } from '../extension.js';
+import { Catch } from '../platform/catch.js';
+import { Store } from '../platform/store.js';
+import { Str } from '../core/common.js';
+import { Composer } from '../composer.js';
+import { ComposerUrlParams } from './interfaces/composer-types.js';
+import { ComposerComponent } from './interfaces/comopser-component.js';
```

Comment:
typo `comopser-component`

Reply:
so many typos, sorry

Reply:
Maybe can install some word-checking plugin. IntelliJ will always underline typos everywhere. There must be something similar for vscode. Then when we add a word to the dictionary of the plugin, the dict file should be check into the repo so everyone can benefit.